scripts/webpack/webpack-resources.js

```diff
@@ -22,25 +22,33 @@ function validateEnv() {
   }
 }
 
+function shouldPrepend(config) {
+  const packageJson = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'package.json'), 'utf-8'));
+  const excludedProjects = ['perf-test', 'test-bundles'];
+  const exportedAsBundle = config.output && (config.output.libraryTarget === 'umd' || config.output.libraryTarget === 'var');
+  const hasReactAsDependency =
+    (packageJson.dependencies && Object.keys(packageJson.dependencies).includes('react')) ||
```

Comment:
Sorry if you answered this on the other PR and I didn't see, but what about peer dependencies?

Reply:
We don't dictate peer deps. So that's just some react implied requirement.

Reply:
Still confused...why does it matter if a package has React as a dep here?